PointOfInteraction/README.md

```diff
@@ -0,0 +1,9 @@
+# Smart Point of Interaction
+
+A point of interaction is composed mainly by to entities:
+ * 1.1 "Smart Point of Interaction": defines an interactive point in a city which provides information, entertainment or co-creation tools to citizens. Optionally can address to an entity which is improved with this technology to interact.
```

Comment:
can reference a related smart city asset with enriched interaction provided by this technology.